src/Page/Food.elm

```diff
@@ -1317,19 +1361,29 @@ mainView session model =
 
 menuView : Query -> Html Msg
 menuView query =
-    div [ class "d-flex gap-2" ]
-        [ button
-            [ class "btn btn-outline-primary"
-            , classList [ ( "active", query == Query.carrotCake ) ]
-            , onClick (LoadQuery Query.carrotCake)
+    let
+        autocompleteState =
+            AutocompleteSelector.init Query.toString Query.recipes
+    in
+    div []
+        [ label
+            [ for "selector-example"
+            , class "form-label fw-bold"
             ]
-            [ text "Carrot Cake" ]
+            [ text "Produit" ]
         , button
-            [ class "btn btn-outline-primary"
-            , classList [ ( "active", query == Query.emptyQuery ) ]
-            , onClick (LoadQuery Query.emptyQuery)
+            [ class "form-select ElementSelector text-start"
+            , id <| "selector-example"
+            , style "overflow" "hidden"
+            , style "white-space" "nowrap"
```

Comment:
As we have applied a custom `ElementSelector` class, shouldn't it rather be updated/extended/specialized to feature these inline styles?